modules/ossm-deploying-jaeger.adoc

```diff
@@ -80,12 +82,42 @@ spec:
           resources: {}
 ----
 
+
+[id="ossm-deploying-jaeger-production_{context}"]
+== Production Jaeger deployment (fully customized)
+
+The SMCP supports only minimal Eleasticsearch parameters.  To fully customize your production environment and access all of the Elasticsearch configuration parameters, use the Jaeger custom resource to configure Jaeger.
+
+Or you can create and configure your Jaeger instance and set  `spec.addons.jaeger.name` to the name of the Jaeger instance, for example  `jaeger-production-cr`.
+
+.Control Plane with linked Jaeger production CR
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+spec:
+  version: v2.0
+  tracing:
+    sampling: 1000
+    type: Jaeger
+  addons:
+    jaeger:
+      name: jaeger-production-cr #name of Jaeger CR
```

Comment:
The CR is just to really clarify that this is a link out to a separate CR, that's all.

Reply:
Alright. I asked only because I can see this being blindly copied by readers, so that they will end up with a CR with "-cr" in the name, which seems like an anti-pattern (like a Deployment named "my-app-deployment").